Source/WebCore/rendering/svg/legacy/LegacyRenderSVGPath.cpp

```diff
@@ -63,9 +64,10 @@ void LegacyRenderSVGPath::updateShapeFromElement()
         m_shapeType = ShapeType::Path;
 }
 
-FloatRect LegacyRenderSVGPath::calculateUpdatedStrokeBoundingBox() const
+FloatRect LegacyRenderSVGPath::calculateUpdatedStrokeBoundingBox(FloatRect strokeBoundingBox) const
 {
-    FloatRect strokeBoundingBox = m_strokeBoundingBox;
+    if (!m_markerPositions.isEmpty())
+        strokeBoundingBox.unite(markerRect(strokeWidth()));
```

Comment:
The code below gets the `strokeWidth()` in a local variable `strokeWidth`. I think we should be consistent and avoid calling this method twice.

Reply:
Changed.